src/test-utils/dom/split-panel/index.ts

```diff
@@ -20,7 +20,8 @@ export default class SplitPanelWrapper extends ComponentWrapper {
   }
 
   findOpenButton(): ButtonWrapper | null {
-    return this.findComponent(`.${styles['open-button']}`, ButtonWrapper);
+    const wrapper = createWrapper();
+    return wrapper.findComponent(`.${styles['open-button']}`, ButtonWrapper);
```

Comment:
do we have a test for the case when it's not inside Split Panel?

Reply:
Yes 
https://github.com/cloudscape-design/components/blob/29aa65765f15ac20a060ee17a6a5f7139159e6fa/src/app-layout/__integ__/app-layout-refresh-edge-controls.test.ts#L87-L89
https://github.com/cloudscape-design/components/blob/8f604c9f1d91153478199064b377b6d8184b4cf0/src/app-layout/__integ__/app-layout-focus-delegation.test.ts#L106-L109